level1c4pps/vgac2pps_lib.py

```diff
@@ -96,58 +98,262 @@
                 "M13": 'ch_tbxx'}
 
 
-def convert_to_noaa19(scene):
-    """
-    Convert channel data to noaa19, using SBAFs.
-
-    # Try I 20230404
-    N19_ch1 = 0.958*M5
-    N19_ch2 = 0.878*M7
-    # N19_ch1 = 1.022*M5 (corr-coeff 0.9995)
-    # N19_ch2 = 0.729*M6 (corr-coeff 0.9987)
-    N19_ch3b = -1.1603*10^2 + 1.8362 * M12 – 1.501*10^3*M12^2 (corr-coeff 0.9942)
-    N19_ch4 = 1.0003*M15 (corr-coeff 1.0!)
-    N19_ch5 = 0.8 + 0.9956*M16 (corr-coeff 1.0!)
-
-    # Try II 20231120
-    r06(AVHRR,%) = 0.9393*M5 + 0.57
-    r09(AVHRR,%) = 0.001275*M7*M7 + 0.7781*M7 +1.77
-    tb37(AVHRR,K) = 0.9563*M12 + 9.86
-    tb11(AVHRR,K) = 1.003*M15 - 0.77
-    tb12(AVHRR,K) = 1.002*M16 - 0.69
-
-    # Try II Nina
-    r06(AVHRR,%) = 0.9034*M5 + 1.04
-    r09(AVHRR,%) = 0.8821*M7 + 0.17
-    tb37(AVHRR,K) = 0.997*M12 + 0.89
-    tb11(AVHRR,K) = 1.001*M15 - 0.05
-    tb12(AVHRR,K) = 0.9899*M16 + 2.5
-
-    r06(AVHRR,%) = 0.914 * M5 + 0.83
-    r09(AVHRR,%) = 0.9019 * M7 - 0.18
-    tb37(AVHRR,K) = 0.9962 * M12 + 0.99
-    tb11(AVHRR,K) = 1.002 * M15 - 0.31
-    tb12(AVHRR,K) = 0.993 * M16 + 1.62
-
-    r06(AVHRR,%) = 0.9211*M5 + 0.85
-    r09(AVHRR,%) = 0.9242*M7 -0.78
-    tb37(AVHRR,K) = 0.9951*M12 + 1.18
-    tb11(AVHRR,K) = 1.003*M15 - 0.55
-    tb12(AVHRR,K) = 0.9977*M16 + 0.39
-
-    KG 20230111:
-    r06(AVHRR, %) = 0.9129 * M5 + 0.99
-    r09(AVHRR, %) = 0.9025 * M7
-    tb37(AVHRR, K) = 0.9967 * M12 + 0.85
-    tb11(AVHRR, K) = 1.002 * M15 - 0.4
-    tb12(AVHRR, K) = 0.9934 * M16 +1.52
-    """
+def convert_to_noaa19(scene, V="v6"):
 
-    scene["M05"].values = 0.9129 * scene["M05"] + 0.99
-    scene["M07"].values = 0.9025 * scene["M07"]
-    scene["M15"].values = 1.002 * scene["M15"] - 0.4
-    scene["M16"].values = 0.9934 * scene["M16"] + 1.52
-    scene["M12"].values = 0.9967 * scene["M12"] + 0.85
+    """_SBAF dictionary_
+
+    I've put the SBAFs in this dictionary for super clarity when running the script
+    and set up for if we want to run more than on version of SBAF
+
+    """
+    SBAF = {
```

Comment:
move this dictionary outside the function. It should be possible to see the complete function without scrolling.